File: AuraFit-BackEnd/src/main/java/app/aurafitbackend/Beans/ContactInformation.java
package app.aurafitbackend.Beans;

import jakarta.persistence.*;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Embeddable
@NoArgsConstructor
@AllArgsConstructor
@Data
@Builder
public class ContactInformation {

//    private Long id;
    private String firstName;
    private String lastName;
    private String email;
    private String phoneNumber;
    private String street;
    private String city;
    private String postalCode;

}

File: aurafit-frontend/src/Models/ContactInformation.ts


export class ContactInformation {
    id: number;
    firstName: string;
    lastName: string;
    email: string;
    phoneNumber: string;
    street: string;
    city: string;
    postalCode: string;

    constructor(id: number, firstName: string, lastName: string, email: string, phoneNumber: string, street: string, city: string, postalCode: string) {
        this.id = id;
        this.firstName = firstName;
        this.lastName = lastName;
        this.email = email;
        this.phoneNumber = phoneNumber;
        this.street = street;
        this.city = city;
        this.postalCode = postalCode;
    }
}